RemoteDesktop/Form1.Designer.cs

```diff
@@ -58,27 +60,46 @@ private void InitializeComponent()
             this.hideBtn.FlatStyle = System.Windows.Forms.FlatStyle.Flat;
             this.hideBtn.ForeColor = System.Drawing.Color.White;
             this.hideBtn.Location = new System.Drawing.Point(160, 0);
+            this.hideBtn.Margin = new System.Windows.Forms.Padding(0);
             this.hideBtn.Name = "hideBtn";
-            this.hideBtn.Size = new System.Drawing.Size(160, 200);
+            this.hideBtn.Size = new System.Drawing.Size(480, 600);
             this.hideBtn.TabIndex = 1;
             this.hideBtn.Text = "Show Desktop";
             this.hideBtn.UseVisualStyleBackColor = false;
             this.hideBtn.Click += new System.EventHandler(this.hideBtn_Click);
             // 
+            // switchBtn
+            // 
+            this.switchBtn.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom) 
+            | System.Windows.Forms.AnchorStyles.Right)));
+            this.switchBtn.BackColor = System.Drawing.Color.FromArgb(((int)(((byte)(64)))), ((int)(((byte)(64)))), ((int)(((byte)(64)))));
+            this.switchBtn.FlatAppearance.BorderColor = System.Drawing.Color.Gray;
+            this.switchBtn.FlatStyle = System.Windows.Forms.FlatStyle.Flat;
+            this.switchBtn.ForeColor = System.Drawing.Color.White;
+            this.switchBtn.Location = new System.Drawing.Point(640, 0);
+            this.switchBtn.Margin = new System.Windows.Forms.Padding(0);
+            this.switchBtn.Name = "switchBtn";
+            this.switchBtn.Size = new System.Drawing.Size(160, 600);
+            this.switchBtn.TabIndex = 2;
+            this.switchBtn.Text = "Switch screen";
+            this.switchBtn.UseVisualStyleBackColor = false;
+            this.switchBtn.Click += new System.EventHandler(this.button1_Click);

```

Comment:
This should event should be renamed. Probably something like "switchBtn_Click" would be fine

Reply:
Oh, I overlooked this moment. Fixed